core/src/test/scala/akka/persistence/cassandra/query/EventsByPersistenceIdFastForwardSpec.scala

```diff
@@ -33,7 +33,7 @@ object EventsByPersistenceIdFastForwardSpec {
 }
 
 class EventsByPersistenceIdFastForwardSpec
-  extends TestKit(ActorSystem("EventsByPersistenceIdSpec", EventsByPersistenceIdSpec.config))
+  extends TestKit(ActorSystem("EventsByPersistenceIdSpec", EventsByPersistenceIdFastForwardSpec.config))
```

Comment:
so it wasn't a bug? good

Reply:
We didn't have debug logging so I don't know that this was definitely the cause but without it it would not refresh.

I only noticed this as i turned on DEBUG logging in my PR and then it failed again and i got no logs

Reply:
Failed locally for me since so think we may have a bug here